.github/workflows/build_and_publish.yaml

```diff
@@ -0,0 +1,60 @@
+name: Tag repo; build and publish assets
+on:
+  push:
+
+jobs:
+  tag-new-version:
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.tag.outputs.tag }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Tag
+        id: tag
+        uses: salsify/action-detect-and-tag-new-version@v1
+        with:
+          version-command: |
+            cat runner/VERSION
+
+  build-and-publish-package:
+    runs-on: ubuntu-latest
+    name: Build and publish docker image
+    # Only on a tagged release
+    needs: tag-new-version
+    if: needs.tag-new-version.outputs.tag
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Build package
+      run: |
+        python setup.py bdist
```

Comment:
I'll need to double-check this but I _think_ we want `sdist bdist_wheel` here, rather than `bdist`.

Reply:
Yes, it [looks like](https://pythonwheels.com/#pure-wheel) we need to do nothing more than change `bdist` to `sdist bdist_wheel`. If that fails then we need to shove a `pip install wheel` line above, but I have a feeling that's redundant these days.